frontend/packages/dev-console/integration-tests/support/step-definitions/addFlow/create-from-git.ts

```diff
@@ -256,3 +256,19 @@ Then('user is able to see {string} value is selected in TLS termination', (value
 Then('user is able to see {string} value is selected in Insecure traffic', (value: string) => {
   cy.get(gitPO.advancedOptions.routing.insecureTraffic).should('have.text', value);
 });
+
+When('devfile import strategy is disabled', () => {
```

Comment:
Nit: This is what you expect:

```suggestion
Then('devfile import strategy is disabled', () => {
```